std/array.d

```diff
@@ -3173,7 +3174,51 @@ if (isDynamicArray!A)
         }
     }
 
+    /**
+     * Gives a string in the form of `Appender!(A)(data)`.
+     *
+     * Params:
+     *     w = A `char` accepting
+     *     $(REF_ALTTEXT output range, isOutputRange, std, range, primitives).
+     *     fmt = A $(REF FormatSpec, std, format) which controls how the array
+     *     is formatted.
+     * Returns:
+     *     A `string` if `writer` is not set; `void` otherwise.
+     */
+    string toString()() const
+    {
+        // Even though the Appender might represent a string,
+        // make a copy of the data anyway because we don't want
+        // manipulations of the toString result to change the
+        // data of the Appender
+        auto app = appender!string();
+        auto spec = singleSpec("%s");
+        // different reserve lengths because each element in a
+        // non-string-like array uses two extra characters for `, `
+        static if (isSomeString!A)
+            app.reserve(_data.arr.length + 15);
+        else
+            app.reserve((_data.arr.length * 3) + 15);
```

Comment:
Hmm, I assume this is the conservative estimate that of `X, ` being three characters which does make sense, though I think it would be nice to add a comment for future readers about this conservative estimate.